atlasdb-client/src/main/proto/com/palantir/atlasdb/protos/TableMetadataPersistence.proto

```diff
@@ -12,6 +12,7 @@ message TableMetadata {
     optional SweepStrategy sweepStrategy = 9;
     optional int32 explicitCompressionBlockSizeKiloBytes = 10;
     optional bool appendHeavyAndReadLight = 11;
+    optional bool nameLoggable = 12 [default = false];
```

Comment:
Is the optional here to handle cases where the metadata was persisted before this version (and so doesn't have `nameLoggable`)?